### ISSUE TITLE
Docker Hub Image

### ISSUE THREAD

**Issue by kanjieater** [slhck/ffmpeg-normalize#260]
**:warning: Please read this carefully and edit the example responses! If you do not fill out this information, your feature request may be closed without comment.**

**Is your feature request related to a problem? Please describe.**
Just a request to have a docker hub image so users don't have to pull the source and build manually. 

**Describe the solution you'd like**
To pull via docker w/o downloading the repo.

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by slhck** [comment]
This is a good suggestion. I never set this up before, but I will give it a try in the next few days.

**Comment by slhck** [comment]
If you want you can already try the `master` build from here: https://hub.docker.com/repository/docker/slhck/ffmpeg-normalize/tags

I am not sure how to set up a `latest` pointer, and I have not yet released a new tag, so I don't know if the automatic workflow will work, but we'll see.

**Comment by slhck** [comment]
Well, there you go, `latest` works too, now. I updated the README to provide more useful Docker instructions. Thanks for bringing this up; I never had a good use case for running a build like this via CI, but it was way easier than I thought.

**Comment by kanjieater** [comment]
That was quick thank you. I played around with my own as well yesterday. 

I used the popular FFMPEG docker image that already had libfdk-aac in & tried to set some reasonable defaults to slim down what actually needs to be used to run and normalize spoken text. 

https://hub.docker.com/repository/docker/kanjieater/ffmpeg-normalize/general

Glad to see one will be  baked into the development process here though. Thanks for the swift work 🙇
